hathor/transaction/transaction.py

```diff
@@ -140,6 +140,10 @@ def calculate_min_height(self) -> int:
             self._calculate_my_min_height(),
         )
 
+    def calculate_feature_activation_bit_counts(self) -> Optional[list[int]]:
+        """Returns None, as Transactions do not have the feature_activation_bit_counts metadata attribute."""
+        return None
```

Comment:
Should we `raise NotImplementedError`?

Reply:
Yes. I had to change the structure a bit to be able to support this, I created an internal, abstract `BaseTransaction._calculate_feature_activation_bit_counts()` that is only implemented by `Block`, and a public `BaseTransaction.calculate_feature_activation_bit_counts()` that is actually implemented in `BaseTransaction` to calls the block's implementation.

Done in https://github.com/HathorNetwork/hathor-core/commit/8a97f4805e9aab445d6a0f5c99df44dcf335f97f

Reply:
Ok. Thanks!